guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,146 @@
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+<slot name="medicalrecord/records-list-headline"></slot>
+<slot name="medicalrecord/records-list-table"></slot>
+```
+
+The layout from MedicalRecord is appended at the end of the PetList wrapper. To blend it, we will move the MedicalRecord table and headline into the `div class="pet-list-wrapper"` and expand the width of the wrapper to fit the table:
+
+```html
+<style>
+    @import '/PetList/style.css';
+    .pet-list-wrapper {
+        max-width: 750px;
+    }
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="medicalrecord/records-list-headline"></slot>
+    <slot name="medicalrecord/records-list-table"></slot>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+```
+
+We have now produced the result shown in the image above; the view from the MedicalRecord app has been neatly integrated with the view from PetList.
+
+## Creating Layouts
+
+The tool for creating these layouts is the [CompositionEditor](https://github.com/starcounterapps/CompositionEditor). It's a Starcounter app that can be opened att any page with `Ctrl + E`. There are two main concepts in the CompositionEditor - the identifier and layout. 
+
+![Composition editor](/assets/CompositionEditor.png)
+
+The identifier is created each time a composition is merged with server-side blending. For example, the identifier for the PetList and MedicalProvider example looks like this:
+
+```
+[partial-id="/sc/htmlmerger?PetList=/PetList/views/PetDetails.html&MedicalRecordProvider=/MedicalRecordProvider/views/RecordsList.html"]
+```
+
+The `/sc/htmlmerger` is a prefix that is added every time a view contains views that come from more than one response. In the identifier above, it's the merged views of `PetDetails.html` and `RecordsList.html`. 
+
+The layout changes made in the editor are displayed in real time.
+
+Layouts are saved in the database and can be queried for with `SELECT * FROM Starcounter.HTMLComposition`. 
+
+## Providing Layouts
+
+Layouts are provided by the [CompositionProvider](https://github.com/starcounterapps/compositionprovider), another Starcounter app. When responses are merged, it looks if there's a layout in the database with the same identifier and serves it, otherwise, the default layout is served. 
+
+Because of this, the CompositionProvider has to be running when client-side blending should be used. 
```

Comment:
Maybe we could a add similar not for the editor - that it's just a dev tool, that could be stopped on production env.